prime-dotnet-webapi/Sentry/ISentryErrorReporter.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.Threading.Tasks;
+
+namespace SentryCustomReporter
+{
```

Comment:
Although capturing these exceptions is definitely useful, not sure if this warrants its own folder, i.e. `Sentry`, and namespace?

Reply:
Good point. I only did that because I wasn't sure which namespace to use for it and which directory. Originally I put it in **prime-dotnet-webapi** root, but then it seemed like a lot of files in root so I moved it to its own directory. Will try to find a better home for it

Reply:
Now Sentry is part of the Prime.Helpers name space and mopved the files to /Helpers directory